src/develop/blend.c

```diff
@@ -2644,7 +2644,7 @@ void dt_develop_blend_process(struct dt_iop_module_t *self, struct dt_dev_pixelp
     }
 
 #ifdef _OPENMP
-#if !defined(__SUNOS__) && !defined(__NetBSD__) && !defined(__WIN32__)
+#if !defined(__SUNOS__) && !defined(__NetBSD__) && !defined(__WIN32__) && defined(__GLIBC__)
```

Comment:
Compiler complaining about stderr on line 2648. Why is stderr there? It's not used in the for loop! If I remove stderr it compiles just fine also on Fedora 23